robot/rover/MobilePlatform/MobilePlatform.ino

```diff
@@ -200,45 +234,255 @@ void roverVelocityCalculator(void) {
   Helpers::get().println(" m ^ 2 / 6 ");
 }
 
-//! Figures out which serial being used
+// Acquire method based on command sent
 void serialHandler(void) {
-  if (devMode) {
-    if (Serial1.available()) {
-      cmd = Serial1.readStringUntil('\n');
-      cmd.trim();
-      Helpers::get().ser_flush();
-      if (cmd == "who") {
-        devMode = false ;
-      }
-      Cmds.handler(cmd, "UART");
+  if (Serial.available()) {
+    // Pointer to select the method (WheelsCommandCenter.cpp) to run based on the command
+    internal_comms::readCommand(commandCenter); 
+  }
+}
```

Comment:
I think this method is kinda useless, you may as well put this directly in loop()